scripts/main.js

```diff
@@ -71,4 +71,91 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+console.log(departments);
+// ------- opdracht 1 -----------
+
+//Opdracht 1a:
+console.log("De afdeling Sales heeft ["+ departments.sales.numberOfEmployees +"] medewerkers");
+
+
+//Opdracht 1b:
+console.log("Marketing is eenleuke afdeling om te werken. " + departments.marketing.description);
+
+
+//opdracht 1c:
+console.log("De afdeling Customer Service heeft " + departments["customer-service"].numberOfEmployees + " medewerkers");
+
+
+//Opdracht 1d:
+console.log("Sales is een uitdagende afdeling om te werken als Verkoopmanager. " + departments.sales.jobs[1].description);
+
+
+
+// ------- opdracht 2 -----------
+//
+// const userInput = prompt('Hoi! Hoe heet je?');
+// console.log(userInput);
+const departmentUserInput = prompt("Over welke afdeling wil je meer informatie? Kies uit: [marketing / sales / customer-service]");
+// // prompt(`Je koos[${userInput}]. ${departments.userInput.description}`);
+//
+switch(departmentUserInput) {
+    case 'marketing':
+        console.log("Je koos " + departmentUserInput + ". " + departments.marketing.description);
+        break;
+    case 'sales':
+        console.log("Je koos " + departmentUserInput + ". " + departments.sales.description);
+        break;
+    case 'customer-service':
+        console.log("Je koos " + departmentUserInput + ". " + departments['customer-service'].description);
+        break;
+    default:
+        console.error("Ongeldige keuze. Porbeer het opnieuw door de pagina te verversen.");
+        break;
```

Comment:
Deze break is niet perse nodig. Maar kan wel natuurlijk. Als je constitent wil zijn. Zet dit ook in je andere swich case.